src/lib/fetcher.ts

```diff
@@ -4,8 +4,9 @@ import { RequestOptions } from './requestOptions';
 import * as querystring from 'querystring';
 import events = require('events');
 import urlJoin = require('url-join');
-
-import fetch = require('isomorphic-fetch');
+if (navigator.product !== 'ReactNative') {
```

Comment:
Yeah, 0.0.12/13 throw this exception: https://github.com/kidtsunami/fetch-salesforce/issues/16